parser/parser.go

```diff
@@ -208,6 +208,134 @@ func parseUnary(p *Parser) (ast.Node, error) {
 	return ast.NewUnaryExpr(tok.Type, expr), nil
 }
 
+func parseIdentExpr(p *Parser) (ast.Node, error) {
+	tok := p.lookahead[0]
+	p.scry(1)
+	next := p.lookahead[1]
+
+	// eg.: console.
+	if next.Type == token.Dot {
+		p.forget(1)
+		return parseMemberExpr(p, ast.NewIdent(tok.Value))
+	}
+
+	// eg.: console(
+	if next.Type == token.LParen {
```

Comment:
Same here, something like "isFunctionCall" may render the comment useless

Reply:
the next token being an LParen does not mean it's function call. It depends on the current context. As we have more JS features implemented you'll see that this is convoluted, because as an example, `for(` isn't a funcall and, in other way, `}()` is.